recipes/wxsqlite3/all/conanfile.py

```diff
@@ -0,0 +1,128 @@
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def requirements(self):
+        # https://github.com/utelle/wxsqlite3/blob/v4.10.8/premake/wxwidgets.lua#L146
+        self.requires("wxwidgets/[>=3.2.5 <3.3]", transitive_headers=True, transitive_libs=True)
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.tool_requires("premake/5.0.0-beta7")
+        else:
+            self.tool_requires("libtool/2.4.7")
+
+    def generate(self):
+        if self.settings.os == "Windows":
+            deps = PremakeDeps(self)
+            deps.generate()
+            tc = PremakeToolchain(self)
+            tc.generate()
+        else:
+            wxwidgets_root = self.dependencies["wxwidgets"].package_folder
+            wx_config = os.path.join(wxwidgets_root, "bin", "wx-config")
+            tc = AutotoolsToolchain(self)
+            tc.configure_args.append(f"--with-wx-config={wx_config}")
+            tc.generate()
+            deps = AutotoolsDeps(self)
+            deps.generate()
+
+    def build(self):
+        if self.settings.os == "Windows":
+            premake = Premake(self)
+            premake.configure()
+            platform = self._arch_to_msbuild_platform(self.settings.arch)
+            premake.build(workspace=f"wxsqlite3_{self._msvc_version_str()}", targets=["wxsqlite3"], msbuild_platform=platform)
+        else:
+            autotools = Autotools(self)
+            autotools.autoreconf()
+            autotools.configure()
+            autotools.make()
+
+    def package(self):
+        if self.settings.os == "Windows":
+            copy(self, "*", os.path.join(self.source_folder, "include"), os.path.join(self.package_folder, "include"))
+            lib_dir = os.path.join(self.source_folder, "lib")
+            subdirs = [d for d in os.listdir(lib_dir) if os.path.isdir(os.path.join(lib_dir, d))]
+            copy(self, "*", os.path.join(lib_dir, subdirs[0]), os.path.join(self.package_folder, "lib"))
+        else:
+            autotools = Autotools(self)
+            autotools.install()
+            rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+            rm(self, "*.la", os.path.join(self.package_folder, "lib"))
+
+        license_files = [
+            "COPYING.txt",
+            "GPL-3.0.txt",
+            "LGPL-3.0.txt",
+            "LICENSE.spdx",
+            "LICENSE.txt",
+            "WxWindows-exception-3.1.txt",
+        ]
+
+        for license_file in license_files:
+            copy(self, license_file, dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+
+    def package_info(self):
+        if self.settings.os == "Windows":
+            self.cpp_info.libs = ["wxsqlite3"]
+        if self.settings.os == "Macos":
```

Comment:
This is the fix for my previous comment

```suggestion
        elif self.settings.os == "Macos":
```